test/spec/aggregate.test.ts

```diff
@@ -10,13 +10,14 @@ interface IAggregateSigsTestCase {
     input: string[];
     output: string;
   };
+  meta?: undefined;
 }
 
 describeForAllImplementations((bls) => {
   describeDirectorySpecTest<IAggregateSigsTestCase, string | null>(
     "bls/aggregate/small",
     path.join(SPEC_TESTS_DIR, "tests/general/phase0/bls/aggregate/small"),
-    (testCase) => {
+    (testCase: any) => {
```

Comment:
do you know why this worked previously, ideally we should make sure to use proper types